rfc/0009-audit-checks.md

```diff
@@ -137,82 +120,52 @@ spec:
     - CREATE
     - UPDATE
   mutating: true
-  audit: true
+  backgroundAudit: true
   settings:
     allowed_capabilities:
     - CHOWN
     required_drop_capabilities:
     - NET_ADMIN
 ```
 
-## Policy server and policies communication
-
-To be able to call policies to evaluate the resources already in the cluster,
-it's necessary to add a new fields in the `ValidationRequest` (e.g. `resources`)
-and `ValidationRespose` (e.g. `resources_validations`). In the `ValidationRequest`
-the field is needed to store  a list of the resources which the policy must validate.
-And in the `ValidationResponse` is necessary a field to store a list of the results
-of the resources validations given in the `ValidationRequest` type.  This way the
-policies will have a unified communication interface.  With no need to implement
-additional functions. However, the policies still need to be updated to verify if
-this field is populated and, if it is, verify the resources there.
-
-Furthermore, the `policy-evaluator` should also be updated to propagate this field
-until the Policy Server where the audit checks will be trigger and the results will be processed.
-For that, the types used to store the policies responses as well as the functions used
-to process these types and to send the validation request to the policies, must be updated
-to add the new fields used to store the audit checks resources and results.
-
-## Audit check results
+# Drawbacks
 
-Once the policy server has the audit checks results, it will populate the a [PolicyReport](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report).
-The report will contains all the success and failures  returned by the policies.
-Results from `ClusterAdmissionPolicy` (cluster wide policies) will be stored in the `ClusterPolicyReport`
-and the result from the `AdmissionPolicy` (namespaces policies) will be stored in the `PolicyReport`.
+Both Audit Background Scanner and Policy Server must write to the same PolicyReport. It might happen that both try to
```

Comment:
We may have more than 1 PolicyServer trying to write. Hence I favour alternative A, Auditor Scanner having a rest API, and being the only one that can add PolicyReportResults.